include/mitsuba/render/bsdf.h

```diff
@@ -519,6 +519,23 @@ class MI_EXPORT_LIB BSDF : public Object {
     virtual Spectrum eval_diffuse_reflectance(const SurfaceInteraction3f &si,
                                               Mask active = true) const;
 
+    /**
+     * \brief Evaluate roughness
+     *
+     * This method approximates the roughness for a given
+     * direction. For some materials, an exact value can be computed
+     * inexpensively.
+     * When this is not possible, the value is approximated by
+     * evaluating the BSDF for a normal outgoing direction and returning this
+     * value multiplied by pi. This is the default behaviour of this method.
```

Comment:
Please update this :smile:

Reply:
Why shouldn't it be exactly the same? doesn't it under the hood call same methods as diffuse_reflectance_eval?

Reply:
I'm confused. 

Doesn't the default behaviour just return 1 currently? It doesn't use the BSDF evaluation at the normal direction.